addons/website/views/snippets/s_quotes_carousel_minimal.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_quotes_carousel_minimal" name="Quotes Minimal">
+    <section class="s_quotes_carousel_wrapper" data-vxml="001" data-vcss="002">
+        <t t-set="uniq" t-value="datetime.datetime.now().microsecond"/>
+        <div t-attf-id="myQuoteCarouselMinimal{{uniq}}" class="s_quotes_carousel_minimal s_carousel_default carousel carousel-dark slide" data-bs-interval="10000">
```

Comment:
Also here, for the same reason as `s_quotes_carousel_wrapper`,  use `s_quotes_carousel` instead of `s_quotes_carousel_minimal` so that you don't need to add selectors in the options

Reply:
Ok 👍 
I've updated the branch : https://github.com/odoo/odoo/compare/59a817675806d9063f1aed77213ec58b38992086..f23ab0f75e32b37fa6548fcde8a9f1904a4cc7db
What do you think ?